### PR TITLE
small fixes for unit tests

### PR DESCRIPTION
**Pull request by thinkingmedia**
One of the tests were failing for me on Windows (stupid line feeds).

While looking at the unit tests. PhpStorm was flagging various things with warnings. So I added definitions for private properties and fix a couple of grammars.

I noticed that there aren't any units for the widgets. Can widgets be tested? How would that be done?

### CODE REVIEW

**Comment by thinkingmedia** [comment]
never mind, I found the widget tests from cakephp. Doesn't look hard. I'll see if I can add a few.

File: tests/TestCase/View/Helper/FlashHelperTest.php [hunk 1]
```diff
@@ -15,6 +14,15 @@
  */
 class FlashHelperTest extends TestCase
 {
+    /**
+     * @var View
+     */
+    private $View;
```
Comment (ADmad) [inline tests/TestCase/View/Helper/FlashHelperTest.php]: We don't use privates and protected would need the var name to be prefixed with underscore. So just change all to `public`.

File: tests/TestCase/View/Helper/PaginatorHelperTest.php [hunk 2]
```diff
@@ -50,6 +59,7 @@ public function setUp()
         Router::connect('/:controller/:action/*');
         Router::connect('/:plugin/:controller/:action/*');
 
+        // @todo Set locale for unit test without using private property.
```
Comment (ADmad) [inline tests/TestCase/View/Helper/PaginatorHelperTest.php]: Todos almost always get left forever :)  Either fix the issue or remove the todo.

File: tests/TestCase/View/Helper/FlashHelperTest.php (hunk 1 above)
Comment (thinkingmedia) [inline tests/TestCase/View/Helper/FlashHelperTest.php]: Question: Why don't you use privates?

**Comment by thinkingmedia** [comment]
am I suppose to post a comment to say I did the changes or does GitHub somehow notify people that I made the changes? Just wondering how automated this PR stuff is.

**Comment by ADmad** [comment]
GH doesn't notify when new commits are added to PR. So a comment is fine.

**Pull request merged**